mino/minogrpc/server.go

```diff
@@ -187,7 +187,7 @@ func (rpc RPC) Stream(ctx context.Context,
 
 		header := metadata.New(map[string]string{
 			headerURIKey:     rpc.uri,
-			headerAddressKey: ""})
+			headerAddressKey: addr.String()})
```

Comment:
Can you add a comment to explain what is this address ?

Reply:
When a server wants to send a message to someone it doesn't know, it then creates an enveloppe with the "from" and "to" and sends it back. However, a server does not know its own address, this is why we set it in the header metadata.

If I make a better use of the enveloppe messages, I should be able to get rid of this attribute (the sever could just read the "to" attribute of the enveloppe to know its own address).

Reply:
I'm not sure to understand why a server couldn't know it's own address. Can't you instantiate the service with the address given when instantiating mino ?

Reply:
that's definitely doable!